tests/instances/precompiles/src/lib.rs

```diff
@@ -6402,3 +6402,72 @@ fn bench_modexp() {
         .expect("Tx should have succeeded");
     }
 }
+
+#[test]
+fn test_regression_p256_is_warm() {
+    let mut chain = rig::Chain::empty(None);
+    let wallet = chain.random_signer();
+    let target = Address::from_slice(
+        hex::decode("0000000000000000000000000000000000000100")
+            .unwrap()
+            .as_slice(),
+    );
+    let forwarder = address!("0x1000000000000000000000000000000000000000");
+
+    chain.set_balance(
+        B160::from_be_bytes(wallet.address().into_array()),
+        U256::from(1_000_000_000_000_000_u64),
+    );
+    chain.set_evm_bytecode(
+        B160::from_be_bytes(forwarder.into_array()),
+        &hex::decode(FORWARDER_BYTECODE).unwrap(),
+    );
+
+    // Just enough for tx to succeed if the address is warm
```

Comment:
It will break after Pectra changes, right? Maybe it would be better to have a special contract which will return gas used specifically by the call to 0x100? It will be more robust and we will have less issues later

Reply:
It should be there, but I didn't check